src/pages/_app.tsx

```diff
@@ -66,15 +67,15 @@ function MainApp(props: any) {
 
 // @ts-ignore
 MainApp.getInitialProps = async (appContext: any) => {
-    // const { ctx } = appContext;
+    const { ctx } = appContext;
     const appProps = await App.getInitialProps(appContext);
-    // let userAgent;
-    // // @ts-ignore
-    // if (process && process.browser) {
-    //     userAgent = navigator.userAgent;
-    // } else {
-    //     userAgent = ctx.req.headers['user-agent'];
-    // }
+    let userAgent;
+    // @ts-ignore
+    if (process && process.browser) {
+        userAgent = navigator.userAgent;
+    } else {
+        userAgent = ctx.req.headers['user-agent'];
+    }
```

Comment:
Comment out these lines for now, this place needs proper refactoring. I'll handle this later